docs/designdoc/support_reduce_volume_size.md

```diff
@@ -0,0 +1,153 @@
+
+It is performed as follows:
+
+```console
+$ kubectl delete statefulset <statefulset-name> --cascade=orphan
+```
+
+Subsequently, the moco-controller recreates the StatefulSet with the new `.spec.volumeClaimTemplates`.
+
+### 3. The user manually deletes the MySQL Pod & PVC
+
+The user manually deletes the PVC and Pod.
+Use the following command to delete them:
+
+```console
+$ kubectl delete --wait=false pvc <pvc-name>
+$ kubectl delete --grace-period=1 <pod-name>
+```
+
+### 4. Wait for the Pod & PVC to be recreated by the statefulset-controller, and for MOCO to clone the data
+
+The statefulset-controller recreates Pods and PVCs, creating a new PVC with a reduced size.
+Once the MOCO successfully starts a Pod, it begins cloning the data.
+
+```console
+$ kubectl get mysqlcluster,po,pvc
+NAME                                AVAILABLE   HEALTHY   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   LAST BACKUP
+mysqlcluster.moco.cybozu.com/test   True        False     0         2                                   <no value>
+
+NAME              READY   STATUS     RESTARTS   AGE
+pod/moco-test-0   3/3     Running    0          2m14s
+pod/moco-test-1   3/3     Running    0          114s
+pod/moco-test-2   0/3     Init:1/2   0          7s
+
+NAME                                           STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/mysql-data-moco-test-0   Bound    pvc-03c73525-0d6d-49de-b68a-f8af4c4c7faa   1Gi        RWO            standard       2m14s
+persistentvolumeclaim/mysql-data-moco-test-1   Bound    pvc-73c26baa-3432-4c85-b5b6-875ffd2456d9   1Gi        RWO            standard       114s
+persistentvolumeclaim/mysql-data-moco-test-2   Bound    pvc-779b5b3c-3efc-4048-a549-a4bd2d74ed4e   500Mi      RWO            standard       7s
+```
+
+### 5. Once the cluster becomes Healthy, the user deletes the next Pod and PVC
+
+The user waits until the MySQLCluster state becomes Healthy, and then deletes the next Pod and PVC.
+
+```console
+$ kubectl get mysqlcluster
+NAME                                AVAILABLE   HEALTHY   PRIMARY   SYNCED REPLICAS   ERRANT REPLICAS   LAST BACKUP
+mysqlcluster.moco.cybozu.com/test   True        True      1         3                                   <no value>
+```
+
+### 6. It is completed when all Pods and PVCs are recreated
+
+Repeat steps 3 to 5 until all Pods and PVCs are recreated.
+
+### Metrics
+
+As with volume expansion, the number of times a StatefulSet is recreated and the number of errors are provided as metrics.
+
+```text
+moco_cluster_statefulset_recreate_total{name="mycluster", namespace="mynamesapce"} 3
+moco_cluster_statefulset_recreate_errors_total{name="mycluster", namespace="mynamesapce"} 1
+```
+
+If a StatefulSet fails to recreate, the metrics in `moco_cluster_statefulset_recreate_errors_total` is incremented after each reconcile,
+so users can notice anomalies by monitoring this metrics.
+
+### Conditions
```

Comment:
As commented verbally, this status is not necessary.

In the volume reduction steps, the scope of moco's responsibility is only step 2 (update StatefulSet).
Other steps (deleting pods, etc.) are the user's responsibility.
So there is no need to manage it as status.

Reply:
Thanks for the feedback, I'll try to get it back to you.
I believe the description of the condition is gone now, so please review it again 🙏